ALU_wrapper.py

```diff
@@ -0,0 +1,128 @@
+#!/apps/bio/software/anaconda2/envs/hcp/bin/python3
+
+# Please note, this is an early version. All the verbose-stuff will be reduced at later stages.
+# TODO: functions for copy finished data to web folder
+# TODO: Information to slims?
+# TODO: What to do with original folders?
+
+import os
+import glob
+import smtplib
+import argparse
+import subprocess
+from email.message import EmailMessage
+
+mail_to = ""  # Change this, and uncomment argument before EOF
+pythonpath = "/apps/bio/software/anaconda2/envs/hcp/bin/python3"  # since server (as for now) has python2 installed
+
+parser = argparse.ArgumentParser(description='Wrapper to find non-processed folders, run ALU_parser on them.')
+parser.add_argument("-p", "--path", help="Path to folder with sequence-subfolders.", action="store", default=".")
+# /seqstore/remote/share/crc
+parser.add_argument("-r", "--run", help="Run for real. If not specified, just dry-run without any changes.",
+                    action="store_true")
+parser.add_argument("-v", "--verbose", help="Show more information while running.", action="store_true")
+args = parser.parse_args()
+sendmail = False
+seqpath = args.path
+
+
+def email_general(msg_to, msg_from, subject, body, attachment=None):
+    msg = EmailMessage()
+    msg.set_content(f'{body}\n'
+                    f'\n'
+                    f'Kind regards,\n'
+                    f'Clinical Genomics Gothenburg')
+    msg['Subject'] = f'{subject}'
+    msg['From'] = msg_from
+    msg['To'] = msg_to
+    # msg['Cc'] = "clinicalgenomics@gu.se"
+    if attachment:  # Add attachment if provided
+        csv_filename = os.path.basename(attachment)
+        with open(attachment, 'rb') as f:
+            data = f.read()
+            msg.add_attachment(data, maintype='text', subtype='plain', filename=csv_filename)
+    s = smtplib.SMTP('smtp.gu.se')
+    s.send_message(msg)
+    s.quit()
```

Comment:
Add CGG repo as submodule and import from there

Reply:
As this is going to be used as a cron-script, do we have that repo in a central place, for scripts to import from? Damn, I have missed to check that... But if we do not have that, we probably *should* have it (?). That would be nice for imports, and don't need to exist in several different places for different scripts.

Reply:
Done in latest commit (433c048)